Multiple embedded resources issue

Using a simple starter template:

```xml
<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netstandard1.6</TargetFramework>
    <Version>1.0.0-alpha</Version>
    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="Library.fs" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
    <PackageReference Include="FSharp.Core" Version="4.1.*" />
  </ItemGroup>
</Project>
```
Adding more than one .resx file to the folder results in the following build error:
```
C:\Program Files\dotnet\sdk\1.0.0-rc4-004771\Microsoft.Common.CurrentVersion.targets(2857,5): error MSB3577: Two output file
 names resolved to the same output path: "obj\Debug\netstandard1.6\.resources" [C:\Projects\test_fs\test_fs.fsproj]
```
P.S. C# templates work fine with multiple .resx

--- a/build.proj
+++ b/build.proj
@@ -30,7 +30,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <FSharpNETSdkVersion>1.0.0</FSharpNETSdkVersion>
+    <FSharpNETSdkVersion>1.0.1</FSharpNETSdkVersion>
     <FSharpSdkVersion>1.0.0</FSharpSdkVersion>
   </PropertyGroup>
 

--- a/src/FSharp.NET.Sdk/build/FSharp.NET.Core.Sdk.targets
+++ b/src/FSharp.NET.Sdk/build/FSharp.NET.Core.Sdk.targets
@@ -21,6 +21,8 @@ this file.
 
     <!--TODO error if FscTaskAssemblyPath is not defined-->
     <UsingTask TaskName="Fsc" AssemblyFile="$(FscTaskAssemblyPath)" />
+    <!--TODO add property CreateFSharpManifestResourceNameTaskAssemblyPath -->
+    <UsingTask TaskName="CreateFSharpManifestResourceName" AssemblyFile="$(FscTaskAssemblyPath)"/>
 
     <PropertyGroup>
         <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
@@ -52,18 +54,64 @@ this file.
         <_ExplicitReference Include="$(FrameworkPathOverride)\mscorlib.dll" />
     </ItemGroup>
 
-    <!--Required Tasks by .Common.target-->
-
+    <!--
+      The CreateManifestResourceNames target create the manifest resource names from the .RESX
+      files.
+      
+          [IN]
+          @(EmbeddedResource) - The list of EmbeddedResource items that have been pre-processed to add metadata about resource type
+                                Expected Metadata "Type" can either be "Resx" or "Non-Resx"
+          [OUT]
+          @(EmbeddedResource) - EmbeddedResource items with metadata         
+          
+      For F# applications the transformation is like:
+          Resources1.resx => Resources1 => Build into main assembly
+          SubFolder\Resources1.resx => SubFolder.Resources1 => Build into main assembly
+          Resources1.fr.resx => Resources1.fr => Build into satellite assembly
+          Resources1.notaculture.resx => Resources1.notaculture => Build into main assembly
+      For other project systems, this transformation may be different.
+      -->
+    <!-- target CreateManifestResourceNames is required by .Common.target-->
     <PropertyGroup>
         <CreateManifestResourceNamesDependsOn></CreateManifestResourceNamesDependsOn>
     </PropertyGroup>
     <Target
         Name="CreateManifestResourceNames"
-        Condition="'@(EmbeddedResource)' != ''"
+        Condition=" '@(EmbeddedResource)' != '' "
         DependsOnTargets="$(CreateManifestResourceNamesDependsOn)"
         >
 
-        <Message Text="Prova CreateManifestResourceNames" />
+        <ItemGroup>
+            <_Temporary Remove="@(_Temporary)" />
+        </ItemGroup>
+
+        <!-- Create manifest names for culture and non-culture Resx files, and for non-culture Non-Resx resources -->
+        <CreateFSharpManifestResourceName
+              ResourceFiles="@(EmbeddedResource)"
+              RootNamespace="$(RootNamespace)"
+              Condition=" '%(EmbeddedResource.ManifestResourceName)' == '' and ( '%(EmbeddedResource.WithCulture)' == 'false' or '%(EmbeddedResource.Type)' == 'Resx' ) ">
+
+            <Output TaskParameter="ResourceFilesWithManifestResourceNames" ItemName="_Temporary" />
+
+        </CreateFSharpManifestResourceName>
+
+        <!-- Create manifest names for all culture non-resx resources -->
+        <CreateFSharpManifestResourceName
+              ResourceFiles="@(EmbeddedResource)"
+              RootNamespace="$(RootNamespace)"
+              PrependCultureAsDirectory="false"
+              Condition=" '%(EmbeddedResource.ManifestResourceName)' == '' and '%(EmbeddedResource.WithCulture)' == 'true' and '%(EmbeddedResource.Type)' == 'Non-Resx' ">
+
+            <Output TaskParameter="ResourceFilesWithManifestResourceNames" ItemName="_Temporary" />
+
+        </CreateFSharpManifestResourceName>
+
+        <ItemGroup>
+            <EmbeddedResource Remove="@(EmbeddedResource)" Condition=" '%(EmbeddedResource.ManifestResourceName)' == '' "/>
+            <EmbeddedResource Include="@(_Temporary)" />
+            <_Temporary Remove="@(_Temporary)" />
+        </ItemGroup>
+
     </Target>
 
   <Target

--- a/test/dotnet-new.Tests/CommonScenarioTests.cs
+++ b/test/dotnet-new.Tests/CommonScenarioTests.cs
@@ -289,5 +289,33 @@ namespace NetcoreCliFsc.Tests
                 .Execute($"build {LogArgs}")
                 .Should().Pass();
         }
+
+
+        [Fact]
+        public void TestAppWithRes()
+        {
+            var rootPath = Temp.CreateDirectory().Path;
+
+            TestAssets.CopyDirTo("TestAppWithRes", rootPath);
+            TestAssets.CopyDirTo("TestSuiteProps", rootPath);
+
+            Func<string,TestCommand> test = name => new TestCommand(name) { WorkingDirectory = rootPath };
+
+            test("dotnet")
+                .Execute($"restore {RestoreDefaultArgs} {RestoreSourcesArgs(NugetConfigSources)} {RestoreProps()}")
+                .Should().Pass();
+
+            test("dotnet")
+                .Execute($"build {LogArgs}")
+                .Should().Pass();
+
+            var result = test("dotnet").ExecuteWithCapturedOutput($"run {LogArgs}");
+
+            result.Should().Pass();
+
+            Assert.NotNull(result.StdOut);
+            Assert.Equal("Hi!", result.StdOut.Trim());
+        }
+
     }
 }

--- a/test/testAssets/TestAppWithRes/Messages.resx
+++ b/test/testAssets/TestAppWithRes/Messages.resx
@@ -1,0 +1,124 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
+    
+    The primary goals of this format is to allow a simple XML format 
+    that is mostly human readable. The generation and parsing of the 
+    various data types are done through the TypeConverter classes 
+    associated with the data types.
+    
+    Example:
+    
+    ... ado.net/XML headers & schema ...
+    <resheader name="resmimetype">text/microsoft-resx</resheader>
+    <resheader name="version">2.0</resheader>
+    <resheader name="reader">System.Resources.ResXResourceReader, System.Windows.Forms, ...</resheader>
+    <resheader name="writer">System.Resources.ResXResourceWriter, System.Windows.Forms, ...</resheader>
+    <data name="Name1"><value>this is my long string</value><comment>this is a comment</comment></data>
+    <data name="Color1" type="System.Drawing.Color, System.Drawing">Blue</data>
+    <data name="Bitmap1" mimetype="application/x-microsoft.net.object.binary.base64">
+        <value>[base64 mime encoded serialized .NET Framework object]</value>
+    </data>
+    <data name="Icon1" type="System.Drawing.Icon, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+        <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
+        <comment>This is a comment</comment>
+    </data>
+                
+    There are any number of "resheader" rows that contain simple 
+    name/value pairs.
+    
+    Each data row contains a name, and value. The row also contains a 
+    type or mimetype. Type corresponds to a .NET class that support 
+    text/value conversion through the TypeConverter architecture. 
+    Classes that don't support this are serialized and stored with the 
+    mimetype set.
+    
+    The mimetype is used for serialized objects, and tells the 
+    ResXResourceReader how to depersist the object. This is currently not 
+    extensible. For a given mimetype the value must be set accordingly:
+    
+    Note - application/x-microsoft.net.object.binary.base64 is the format 
+    that the ResXResourceWriter will generate, however the reader can 
+    read any of the formats listed below.
+    
+    mimetype: application/x-microsoft.net.object.binary.base64
+    value   : The object must be serialized with 
+            : System.Runtime.Serialization.Formatters.Binary.BinaryFormatter
+            : and then encoded with base64 encoding.
+    
+    mimetype: application/x-microsoft.net.object.soap.base64
+    value   : The object must be serialized with 
+            : System.Runtime.Serialization.Formatters.Soap.SoapFormatter
+            : and then encoded with base64 encoding.
+
+    mimetype: application/x-microsoft.net.object.bytearray.base64
+    value   : The object must be serialized into a byte array 
+            : using a System.ComponentModel.TypeConverter
+            : and then encoded with base64 encoding.
+    -->
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" />
+    <xsd:element name="root" msdata:IsDataSet="true">
+      <xsd:complexType>
+        <xsd:choice maxOccurs="unbounded">
+          <xsd:element name="metadata">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" />
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Ciao" xml:space="preserve">
+    <value>Hi!</value>
+    <comment>Welcome message</comment>
+  </data>
+</root>

--- a/test/testAssets/TestAppWithRes/Program.fs
+++ b/test/testAssets/TestAppWithRes/Program.fs
@@ -1,0 +1,19 @@
+﻿namespace TestApp
+
+open System
+open System.Reflection
+open System.Resources
+
+type T () = class end
+
+module Program =
+
+    [<EntryPoint>]
+    let main args =
+        try
+            let res = ResourceManager("Messages", (typeof<T>.GetTypeInfo().Assembly))
+            Console.WriteLine(res.GetString("Ciao"))
+            0
+        with ex ->
+            printfn "%A" ex
+            1

--- a/test/testAssets/TestAppWithRes/TestAppWithRes.fsproj
+++ b/test/testAssets/TestAppWithRes/TestAppWithRes.fsproj
@@ -1,0 +1,19 @@
+﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)/TestSuite.props" />
+
+  <ItemGroup>
+    <Compile Include="Program.fs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FSharp.NET.Sdk" Version="$(FSharpNETSdkVersion)" PrivateAssets="All" />
+    <PackageReference Include="FSharp.Core" Version="$(FSharpCorePkgVersion)" />
+  </ItemGroup>
+
+</Project>

